charts/studio/templates/secret-studio.yaml

```diff
@@ -11,14 +11,14 @@ metadata:
     {{- include "studio.labels" . | nindent 4 }}
 type: Opaque
 stringData:
-  {{- if .Values.global.postgres.databaseUrl }}
-  # Deprecated configuration format.
-  # Will be removed in a future release
-  DATABASE_URL: "psql://{{ .Values.global.postgres.databaseUser}}:{{ .Values.global.postgres.databasePassword }}@{{ .Values.global.postgres.databaseUrl }}"
-  {{- else }}
```

Comment:
What if `pgBouncer.enabled` will be `false` ? `DATABASE_URL` will not be set?

How Backend will connect ?

Reply:
I just took an opportunity to remove `DATABASE_URL` because we deprecated it in a previous release. Nowadays, Studio uses separate env vars like `DATABASE_USER`, as you can see further down